validate.py

```diff
@@ -11,6 +11,7 @@
 from collections.abc import Mapping
 from typing import NamedTuple
```

Comment:
this file shouldn't be changed -- validate must succeed on all _installable_ targets.  otherwise we're building a wheel that is installable but broken